src/main/java/net/neoforged/neoforge/common/extensions/IItemExtension.java

```diff
@@ -580,15 +581,14 @@ default boolean canDisableShield(ItemStack stack, ItemStack shield, LivingEntity
 
     /**
      * @return the fuel burn time for this item stack in a furnace. Return 0 to make
-     *         it not act as a fuel. Call super to let the {@link NeoForgeDataMaps#FURNACE_FUELS} decide.
-     * @apiNote This method takes precedence over the {@link NeoForgeDataMaps#FURNACE_FUELS data map}.
+     *         it not act as a fuel. Return -1 to let the default vanilla logic decide.
+     * @apiNote This method takes precedence over the {@link net.neoforged.neoforge.registries.datamaps.builtin.NeoForgeDataMaps#FURNACE_FUELS data map}. 
```

Comment:
Spotless is erroring on the trailing whitespace.
```suggestion
     * @apiNote This method takes precedence over the {@link net.neoforged.neoforge.registries.datamaps.builtin.NeoForgeDataMaps#FURNACE_FUELS data map}.
```